src/app/performence-test/server.js

```diff
@@ -81,15 +83,29 @@ function sendOneByOne() {
     return id;
 }
 
-function createChunck(numOfEntities) {
+function updateChunk(dataArr) {
+    for (let i = 0; i < dataArr.length; i++) {
+        let entity = dataArr[i].entity;
```

Comment:
Wont the entities move in the same direction? Don't we want them to move randomly?